src/main/java/io/jenkins/plugins/file_parameters/StashedFileParameterDefinition.java

```diff
@@ -47,16 +47,11 @@
 
     // TODO equals/hashCode
 
-    @Symbol("stashed64File")
+    @Symbol("stashedBase64File")
```

Comment:
Oh what am I thinking. I meant
```suggestion
    @Symbol("stashedFile")
```